core/validatorapi/router.go

```diff
@@ -348,7 +348,7 @@ func proposeBlock(p eth2client.BeaconBlockProposalProvider) handlerFunc {
 
 		graffiti, err := hexQuery(query, "graffiti")
 		if err != nil {
-			return nil, errors.Wrap(err, "hexadecimal decoding")
+			log.Warn(ctx, "graffiti not present in request query")
```

Comment:
since nothing is wrong, you shouldn't warn. Also, the error might be due to invalid randao in which case you SHOULD error. Rather check if the value is present before trying to decode.